src/connectors/store.py

```
@@ -1,22 +1,110 @@
+STEMMER = Stemmer.Stemmer("spanish")
+
+# Pesos para búsqueda híbrida (0.0 = solo vector, 1.0 = solo BM25)
+BM25_WEIGHT = 0.3
+VECTOR_WEIGHT = 0.7
+
+
+class BM25Retriever(BaseRetriever):
+    """Retriever BM25"""
+    
+    index: Optional[bm25s.BM25] = None
+    documents: List[Document] = []
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)